RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSequence_cff.py

```diff
@@ -0,0 +1,88 @@
+import FWCore.ParameterSet.Config as cms
+
+#==============================================================================
+# Sequence to make low pT electrons.
+# Several steps are cloned and modified to 'open up' the sequence:
+# tracker-driven electron seeds, KF track candidates, GSF tracks.
+#==============================================================================
+
+# Low pT electron seeds
+# Below relies on default configuration for generalTracks
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeeds_cfi import *
+
+# Electron (KF) track candidates
+from TrackingTools.GsfTracking.CkfElectronCandidateMaker_cff import *
+TrajectoryFilterForElectronsOpen = TrajectoryFilterForElectrons.clone()
+TrajectoryFilterForElectronsOpen.minPt = 0.
```

Comment:
@Sam-Harper @mverzett Raising this threshold from 0->1->2 GeV reduces the eff of the Seed->TrkCandidate step (based on generalTracks regardless of match to gen ele) from 98->92->87% (approx!).